js/algoliasearch/internals/frontend/common.js

```diff
@@ -337,11 +337,16 @@ document.addEventListener("DOMContentLoaded", function (e) {
 					var ors = '';
 
 					if (keys.length > 0) {
-						ors += '<span><a href="' + keys[0].url + '">' + keys[0].key + '</a></span>';
-					}
-
-					if (keys.length > 1) {
-						ors += ', <span><a href="' + keys[1].url + '">' + keys[1].key + '</a></span>';
+						var orsTab = [];
+						for (var i = 0; i<algoliaConfig.facets.length; i++) {
+							if (algoliaConfig.facets[i].attribute == "categories") {
```

Comment:
The `if` and `for` to check if categories are present are redundant to `for` on line 324. It's already checked that categories are present. If they wouldn't, `keys` would be an empty array.
So lines 341, 342, 347 (`break;`) can be removed.

At the same time, could you add spaces to `for` arguments?:
```
for (var i = 0; i < keys.length && i < 2 ; i++)
```